jerry-core/ecma/base/ecma-helpers-external-pointers.c

```diff
@@ -119,7 +120,8 @@ ecma_get_external_pointer_value (ecma_object_t *obj_p, /**< object to get proper
     return false;
   }
 
-  JERRY_STATIC_ASSERT (sizeof (uint32_t) <= sizeof (prop_p->u.internal_property.value));
+  JERRY_STATIC_ASSERT (sizeof (uint32_t) <= sizeof (prop_p->u.internal_property.value),
+                       size_of_internal_property_value_must_be_greater_than_or_equal_to_4_bytes);
```

Comment:
This raises the question why this static assert is needed at all? It's a verbatim copy of the static assert above, which makes it useless.